include/cute/arch/mma_xe.hpp

```diff
@@ -35,17 +35,27 @@
 #include <cute/util/sycl_vec.hpp>
 
 #ifdef __SYCL_DEVICE_ONLY__ 
+#ifdef SYCL_INTEL_TARGET
 #define SYCL_DEVICE_OCL(x) SYCL_EXTERNAL x
 #else 
-#define SYCL_DEVICE_OCL(x) inline x { assert(false); }
+#define SYCL_DEVICE_OCL(x)  \
+  inline x { CUTE_INVALID_CONTROL_PATH("Trying to use IGC built-in on non-Intel hardware"); }
 #endif
+#else
+#define SYCL_DEVICE_OCL(x)  \
+  inline x { CUTE_INVALID_CONTROL_PATH("Trying to use device built-in on host."); }
+#endif 
 
 SYCL_DEVICE_OCL(cute::intel::float8 intel_sub_group_bf16_bf16_matrix_mad_k16(cute::intel::short8 a, cute::intel::int8 b, cute::intel::float8 acc));
 SYCL_DEVICE_OCL(float  intel_sub_group_bf16_bf16_matrix_mad_k16(short a, cute::intel::int8 b, float acc));
 #undef SYCL_DEVICE_OCL
 
 namespace cute {
-struct XE_8x16x16_F32BF16BF16F32_TN
+//MxNxK_A,B,C,D
```

Comment:
All the other MMA components in CUTLASS are named using MxNxK_D,A,B,C. We should follow that for consistency. The last two letters in the name indicate whether the data in the A and B inputs is transposed (T) or not (N). See [this link](https://github.com/NVIDIA/cutlass/blob/main/media/docs/cute/0x_gemm_tutorial.md#aside-m-major-n-major-k-major).